app/javascript/mastodon/features/__tests__/toggle-play.jsx

```diff
@@ -34,19 +22,13 @@ class Media extends Component {
         }
       });
     }, 0);
-  };
-
-  render() {
-    const { title } = this.props;
-    const { paused } = this.state;
-
-    return (
-      <button title={title} onClick={this.handleMediaClick}>
-        Media Component - {paused ? 'Paused' : 'Playing'}
-      </button>
-    );
-  }
+  }, [onClick, title]);
```

Comment:
FWIW I personally wouldn't use `useCallback` here. The function is only used by the button a few lines down. But [`react/jsx-no-bind`](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-bind.md) rule is configured to report in this case.